ReactViewResources/Loader/ViewContext.ts

```diff
@@ -0,0 +1,4 @@
+﻿import * as React from "react";
+import { ViewMetadata } from "./ViewMetadata";
+
+export const ViewContext = React.createContext<ViewMetadata>(null!);
```

Comment:
`null!` ?

Reply:
yeah, because the value is mandatory, this is never actually used, but necessary to create a context

Reply:
Didn't know that operator to be used like this 👍